worlds1/WorldBuilder.py

```diff
@@ -80,7 +80,7 @@ def add_agents(builder, condition, task_type, name, folder):
         nr_agents = agents_per_team - human_agents_per_team
         for agent_nr in range(nr_agents):
             if task_type=="official":
-                brain = BaselineAgent(slowdown=8, condition=condition, name=name, folder=folder) # Slowdown makes the agent a bit slower, do not change value during evaluations
+                brain = BaselineAgent(slowdown=1, condition=condition, name=name, folder=folder, trust_mechanism=trust_mechanism) # Slowdown makes the agent a bit slower, do not change value during evaluations
```

Comment:
Why do we make agent slower?

Reply:
we make it faster, I think was mainly to bugtest. @BendikC @Bendikchristensen should know

Reply:
This makes agent faster, I shouldn't actually have commited this so we need to undo it, but for when we're testing it's very handy